src/config/goerli.ts

```diff
@@ -1,19 +1,114 @@
-import { NetworkConfig } from './types';
+import { ZeroAddress } from 'ethers';
+import { Decimal } from '@tempusfinance/decimal';
+import { NetworkConfig, TokenConfig, UnderlyingCollateralTokenConfig } from './types';
+import { getWstEthToStEthRate } from './rates';
+import { Token, UnderlyingCollateralToken } from '../types';
+
+const underlyingTokensConfig: Record<UnderlyingCollateralToken, UnderlyingCollateralTokenConfig> = {
+  wstETH: {
+    address: '0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6',
+    supportedCollateralTokens: {
+      ETH: {
+        address: ZeroAddress,
+        positionManager: '0x4e01f8c03893be67b60af6a1b49d6e51a8781e3c',
+        underlyingCollateralRate: getWstEthToStEthRate,
+        underlyingTokenTicker: 'wstETH',
```

Comment:
Similarly here: we already use `wstETH` as a mapping "root"

Reply:
It makes things easier in some places, when I pass CollateralTokenConfig to some function, I know to which underlying token this collateral belongs to.